inst/extdata/energy/A22.globaltech_co2capture.csv

```diff
@@ -0,0 +1 @@
+# File: A22.globaltech_co2capture.csv
# Title: Primary energy transformation technology capture fractions
# Units: fractions
# Description: Fraction of CO2 captured by transformation technologies
supplysector,subsector,technology,1971,2100
refining,coal to liquids,coal to liquids CCS level 1,0.818,0.818
refining,coal to liquids,coal to liquids CCS level 2,0.981,0.981
refining,biomass liquids,cellulosic ethanol CCS level 1,0.26,0.26
refining,biomass liquids,cellulosic ethanol CCS level 2,0.992,0.992
refining,biomass liquids,FT biofuels CCS level 1,0.818,0.818
refining,biomass liquids,FT biofuels CCS level 2,0.98,0.98

```

Comment:
Please add missing final blank line (here and in any other files that require).

Reply:
these files do have the final return but the formatting is getting broken in however github reads it...it's obviously not a single line (some of the others did not have it). Just an FYI that I checked them all for it, however it's showing up on here.

Reply:
This is a line ending problem then; the files have Windows line endings but we want Unix/Mac. I can fix this, don't worry about it.